### ISSUE TITLE
error on compilation

### ISSUE THREAD

**Issue by loveencounterflow** [axelf4/unicode-linebreak#2]
When trying to use `unicode-linebreak`, I'm getting this error:

```rs
[INFO]: Compiling to Wasm...
   Compiling unicode-linebreak v0.1.1
error[E0277]: the trait bound `&[[u8; 44]; 53]: std::iter::IntoIterator` is not satisfied
   --> /home/flow/.cargo/registry/src/github.com-1ecc6299db9ec823/unicode-linebreak-0.1.1/build.rs:438:16
    |
438 |     for row in &pair_table {
    |                ^^^^^^^^^^^ the trait `std::iter::IntoIterator` is not implemented for `&[[u8; 44]; 53]`
    |
    = help: the following implementations were found:
              <&'a [T; N] as std::iter::IntoIterator>
              <&'a [T] as std::iter::IntoIterator>
              <&'a mut [T; N] as std::iter::IntoIterator>
              <&'a mut [T] as std::iter::IntoIterator>
    = note: required by `std::iter::IntoIterator::into_iter`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0277`.
error: could not compile `unicode-linebreak`.

To learn more, run the command again with --verbose.
Error: Compiling your crate to WebAssembly failed
Caused by: failed to execute `cargo build`: exited with exit code: 101
  full command: "cargo" "build" "--lib" "--target" "wasm32-unknown-unknown"
```

Can you help that?

**Comment by axelf4** [comment]
Version 0.1.1 upped the minimum supported Rust version to 1.47.0. I am sorry for any inconvenience this may have caused. Are you able to update your Rust toolchain, or should I publish a new version lowering the requirement?

**Comment by loveencounterflow** [comment]
Thanks for your swift and precise reply and sorry I had to let you wait for so long; I did
```
rustup update
...
  stable-x86_64-unknown-linux-gnu updated - rustc 1.50.0 (cb75ad5db 2021-02-10) (from rustc 1.45.1 (c367798cf 2020-07-26))
```

and it did compile. I am an absolute beginner in Rust so updating the language was not something I would've thought of in any given amount of time, especially since it doesn't *look* to me like a language version problem. But updating was fast and posed no problems so I guess I'll just have to keep that in mind.
